interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CcdaServiceRequestModelGenerator.php

```diff
@@ -84,11 +84,11 @@ public function create_data($pid, $encounter, $sections, $components, $recipient
             <telecom use='WP' value='" . htmlspecialchars($assignedEntity['telecom'] ?? '', ENT_QUOTES) . "'/>
             <representedOrganization>
                 <name>" . htmlspecialchars($representedOrganization['name'] ?? '', ENT_QUOTES) . "</name>
-                <telecom use='WP' value='" . htmlspecialchars($representedOrganization['telecom'] ?? '', ENT_QUOTES) . "'/>
-                <streetAddressLine>" . htmlspecialchars($representedOrganization['streetAddressLine'] ?? '', ENT_QUOTES) . "</streetAddressLine>
+                <telecom use='WP' value='" . htmlspecialchars($representedOrganization['phone'] ?? '', ENT_QUOTES) . "'/>
```

Comment:
I'm guessing this fixes the phone issues we were running into?

Reply:
yep, you were looking at translate but this never made it there. I was using the wrong telephone template in header author.

Reply:
awesome!  I'm glad you found it!